mlir/test/Dialect/XeGPU/xegpu-wg-to-sg.mlir

```diff
@@ -372,4 +376,72 @@ gpu.func @dpas_no_sg_data(%a: memref<128x128xf16>, %b: memref<128x128xf16>) {
     %cst = arith.constant {layout_result_0 = #xegpu.layout<sg_layout = [8, 4], sg_data = [32, 32]>} dense<1.0> : vector<256x128xf32>
     gpu.return
   }
+
+  // CHECK-LABEL: distribute_load_matrix
+  // CHECK-SAME: [[arg0:%.+]]: memref<32768xi8, 3>
+  gpu.func @distribute_load_matrix(%arg0: memref<32768xi8, 3>) {
```

Comment:
please move the tests to xegpu-wg-to-sg-unify-ops.mlir and xegpu-wg-to-sg-unify-ops-rr.mlir as they are ones that have tests with new versions of create_nd/load_nd/store_nd..... this file will be deleted in the future once we move away from create_nd with offsets

Reply:
Thanks. good to know.